apps/src/lab2/projects/sourcesApi.ts

```diff
@@ -28,7 +28,7 @@ export async function get(
 export async function update(
   channelId: string,
   sources: ProjectSources,
-  options?: SourceUpdateOptions
+  options?: SourceUpdateOptions | {projectType: string | undefined}
```

Comment:
Is the new type sort of like `NewSourceOptions` and `SourceUpdateOptions` extends it?

Reply:
This has been updated according to comment suggestion above. Thanks!